packages/clay-table/src/ClayTable.soy

```diff
@@ -235,13 +239,14 @@
 	]}
 	{@param? _handleCellContentClick: any}
 	{@param? _handleItemToggled: any}
+	{@param? _handleRowContentClick: any}
 	{@param? actionsMenuVariant: string}
 	{@param? groupName: string}
 	{@param? selectable: bool}
 	{@param? showActionsMenu: bool}
 	{@param? spritemap: string}
 
-	<tr>
+	<tr data-onclick="{$_handleRowContentClick}">
```

Comment:
Nice @matuzalemsteles !

Could you please add to this `tr` the class `table-active` when `item.selected` is true?

Thx!